doc/source/cluster/kubernetes/examples/rayjob-kueue-gang-scheduling.md

```diff
@@ -55,14 +55,10 @@ gcloud beta container node-pools create gpu-node-pool \
   --machine-type g2-standard-4
 ```
 
-This command creates a node pool which initially has zero nodes. Use the `gcloud beta` command because some of the flags have beta status.
+This command creates a node pool which initially has zero nodes.
```

Comment:
missing comma:
This command creates a node pool, which initially has zero nodes.